src/main/java/com/expercise/service/cache/CacheService.java

```diff
@@ -0,0 +1,43 @@
+package com.expercise.service.cache;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.core.ZSetOperations;
+import org.springframework.stereotype.Service;
+
+import java.io.Serializable;
+import java.util.Set;
+
+@Service
+public class CacheService {
+
+    @Autowired
+    @Qualifier("objectRedisTemplate")
+    private RedisTemplate<String, Serializable> redisTemplate;
```

Comment:
Is it possible to create RedisTemplate<> childs to prevent using @Qualifier ? It seems error-prone.